game/glshader.cc

```diff
@@ -141,7 +141,7 @@ Shader& Shader::compileCode(std::string const& srccode, GLenum type) {
 	if (gl_response != GL_TRUE) {
 		throw std::runtime_error("Shader compile error.");
 	}
-
+	
```

Comment:
Please setup your editor to remove end-of-line whitespace when the file is saved. Otherwise extra spaces will occur in commits like this here.

Reply:
Remove the extra whitespace please